dired.py

```diff
@@ -1317,24 +1317,28 @@ def __init__(self, view, path):
         self.view.erase_regions('M')
         self.view.erase_regions('?')
         self.call_git(path)
+        self.call_hg(path)
         self.watch_threads()
 
     def watch_threads(self):
         if not 'git' in self.vcs_state:
             sublime.set_timeout(self.watch_threads, 100)
             return
+        if not 'hg' in self.vcs_state:
+            sublime.set_timeout(self.watch_threads, 100)
+            return
```

Comment:
Ah, actually it won’t call twice any way because of return. But still one block instead of two is better, esp. if smb add another VCS in future.